screens/ProfileScreen.tsx

```diff
@@ -1,23 +1,34 @@
 import * as React from 'react';
-import Icon from '../assets/svg-icons/settings.svg';
-import { StyleSheet } from 'react-native';
+import PlusIcon from '../assets/svg-icons/plus-circle.svg';
+import SettingsIcon from '../assets/svg-icons/settings.svg';
+import SearchIcon from '../assets/svg-icons/search.svg';
+import { StyleSheet, TouchableOpacity } from 'react-native';
 import { Text, View } from '../components/Themed';
 
+/* Constants for in TSX inherited properties */
+const pressedOpacity = 0.3;
```

Comment:
just curious, did you choose 0.3 because design said that or just because it looks good?

Reply:
and if this is a global constant that we might use for other touchable opacities, can you add it to constants file?

Reply:
Design said they thought it looked good